react/features/invite/components/DialInNumbersForm.js

```diff
@@ -204,11 +208,18 @@ class DialInNumbersForm extends Component {
     _createDropdownTrigger(triggerText) {
         return (
             <div className = 'dial-in-numbers-trigger'>
-                <input
-                    className = 'input-control'
-                    readOnly = { true }
-                    type = 'text'
-                    value = { triggerText || '' } />
+                <div className = 'form-control__input-container'>
+                    <FieldText
+                        compact = { true }
+                        isLabelHidden = { true }
+                        isReadOnly = { true }
+                        label = 'Select Dial-In Number'
+                        onChange = { _onNoop }
```

Comment:
I did the change. It resulted in some duplication. I could add a wrapper component around FieldText that does a no-op. I'm not sure where I would put that in the project directory, though.